commit 6fc0298185a16e3d04343f8bc818b48d0a4b37df
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   Tue Jun 19 15:40:00 2018 +0000

    change mount option nobootwait to nofail
    
    nobootwait is not supported on debian 8.  nofail is used to allow the system to
    boot far enough that an admin with only ssh access has a chance at fixing it.

diff --git a/cloudinit/config/cc_mounts.py b/cloudinit/config/cc_mounts.py
index 8059011..b9da830 100644
--- a/cloudinit/config/cc_mounts.py
+++ b/cloudinit/config/cc_mounts.py
@@ -77,7 +77,7 @@ def sanitize_devname(startname, transformer, log):
 
 def handle(_name, cfg, cloud, log, _args):
     # fs_spec, fs_file, fs_vfstype, fs_mntops, fs-freq, fs_passno
-    defvals = [None, None, "auto", "defaults,nobootwait", "0", "2"]
+    defvals = [None, None, "auto", "defaults,nofail", "0", "2"]
     defvals = cfg.get("mount_default_fields", defvals)
 
     # these are our default set of mounts
diff --git a/debian/changelog b/debian/changelog
index f9dc56e..c39a49e 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,13 @@
+cloud-init (0.7.6~bzr976-2-joyent.2) unstable; urgency=medium
+
+  * Change default boot options to use nofail instead of nobootwait.
+    - nobootwait is not supported on debian 8
+    - nofail will allow the system to come up into multi-user mode should things
+      go poorly.  Otherwise the VM will be stuck in a state that only the cloud
+      operator can fix.
+
+ -- Mike Gerdts <mike.gerdts@joyent.com>  Tue, 19 Jun 2017 15:38:00 +0000
+
 cloud-init (0.7.6~bzr976-2-joyent.1) unstable; urgency=medium
 
   * Add support for reading DMI data, needed for SmartOS updates
